integration-tests/src/test/java/io/quarkiverse/univocityparsers/it/UnivocityParsersResourceTest.java

```diff
@@ -48,7 +48,7 @@ public void testExampleCsvFile() {
                 "{\"admin\":").append(true).append(",")
                 .append("\"billable\":").append(true).append(",")
                 .append("\"code\":").append("\"DMEADRA-DA\"").append(",")
-                .append("\"createdAt\":").append("\"2015-10-29T23:00:00Z[UTC]\"").append(",")
+                .append("\"createdAt\":").append("\"2015-10-30T00:00:00Z[UTC]\"").append(",")
```

Comment:
You should fix the time zone in your surefire configuration. Otherwise it will break somewhere.